cve-bot.php

```diff
@@ -173,15 +173,6 @@ function fetch_rss($url) {
   $iter = 0;
   while (!$feed && $iter < 5) {
     $iter++;
-/* Commented out because some systems don't have curl.so
```

Comment:
I wanted to leave this in case anyone wanted to use cURL in lieu of file_get_contents().  Maybe a better method would be to add a configuration option (ex. "use_curl = true") and put a conditional here depending on that choice.